conn.go

```diff
@@ -423,6 +425,13 @@ func (s *startupCoordinator) startup(ctx context.Context) error {
 		}
 	}
 
+	for _, ext := range s.conn.cqlProtoExts {
+		var serialized = ext.serialize()
```

Comment:
could use := syntax, var syntax it typically used when you declares variable without a value (nil pointer).